pkgs/servers/u9fs/default.nix

```diff
@@ -1,24 +1,26 @@
 { stdenv, fetchhg }:
 
 stdenv.mkDerivation {
-  name = "u9fs-20110513";
+  pname = "u9fs";
+  version = "unstable-2020-11-21";
+
   src = fetchhg {
-    url = "https://bitbucket.org/plan9-from-bell-labs/u9fs";
-    rev = "9474edb23b11";
+    url = "https://code.9front.org/hg/plan9front";
+    rev = "6eef4d6a9bce";
```

Comment:
Can you use the full hash? Making this hash ambiguous is quite easy.

Reply:
This is the full hash, hg doesn't have a longer one.

Reply:
Just use git ™️